packages/jest-core/src/collectHandles.ts

```diff
@@ -9,10 +9,10 @@
 
 import * as asyncHooks from 'async_hooks';
 import {promisify} from 'util';
-import stripAnsi = require('strip-ansi');
```

Comment:
It seems like `strip-ansi` could be removed from `jest-core/package.json` too. Or?